File: CT-206/Project04/Project04Question01.py
# -*- coding: utf-8 -*-
"""
Created on Tue Apr 10 15:32:54 2018

@author: Nic
"""

f = open ("question1python.txt", "w+")

while True:
    userInput = input("Enter data for file:")
    if userInput == "exit":
        break
    f.write (userInput);

File: CT-206/Project04/Project04Question01.pl
#Project04
#Question01

my $str;
print ("Enter text to write: \n");
print ("Type 'exit' when finished.\n");

while($str ne "exit")
{
  print(">");       
  chomp($str = <STDIN>);  
  
  open(DATA, ">>question1.txt") or die "Can't open file";
  if($str ne "exit")
	{print DATA $str;}
  close(DATA);
}

print ("Write Successful!\n");